### PR TITLE
Create Project within OpenShift

### PR DESCRIPTION
**Pull request by cnuland**
Use the project client within OpenShift to create a new Project based on the definition of the CR file
Require review from @sabre1041 before merge

### CODE REVIEW

**Comment by gnekic** [comment]
Looks solid to me! @cnuland Have you updated trello based on the TODO's that you left?

File: README.md [hunk 1]
```diff
@@ -4,3 +4,37 @@ Project Initialize Operator
...
+```
+$ oc apply -f deploy/crds/redhatcop.redhat.io_projectinitializes_crd.yaml
+$ oc apply -f deploy/service_account.yaml
+$ oc apply -f deploy/role.yaml
+$ oc apply -f deploy/role_binding.yaml
+```
+
+Login to the cluster via the Service Account above
+```
+$ oc sa get-token project-initialize
```
Comment (sabre1041) [inline README.md]: Save the output into a variable. Such as 

```
TOKEN=$(oc sa get-token project-initialize)
oc login --token="${TOKEN}"
```

File: README.md [hunk 2]
```diff
@@ -4,3 +4,37 @@ Project Initialize Operator
...
+$ oc login --token="{above_token}"
+```
+
+Run Operator-SDK
+```
+$ operator-sdk up local --namespace="project-operator"
```
Comment (sabre1041) [inline README.md]: Latest `operator-sdk` uses the following command

```
operator-sdk run --local --namespace="project-operator"
```

**Review by sabre1041 — CHANGES_REQUESTED** [review]
@cnuland A few comments inline but some additional comments

1. Please remove the binary artifact in `build/bin` directory
2. For some of the packages, do we want to adopt the following structure as it allows for different types of implementations?
       1. ldap -> identity
       2. git -> scm

Also be sure to keep in mind some of the patterns and thoughts in [onboarding-manager](https://github.com/redhat-cop/onboarding-manager)

File: README.md (hunk 2 above)
Comment (cnuland) [reply README.md]: done

File: README.md (hunk 1 above)
Comment (cnuland) [reply README.md]: done

**Review by sabre1041 — APPROVED** [review]
LGTM

**Pull request merged**